### PR TITLE
PLT-520 Standardize django-select2-forms

### PR DESCRIPTION
**Pull request by lolasov**
ticket: https://svalbard.atlassian.net/browse/PLT-520

- replaced Travis with GH Actions
- updated README
- got rid of `six`
- updated supported Python/Django versions
- replaced `runtests.py` with `pytest`

### CODE REVIEW

**Comment by lolasov** [comment]
@dakrauth this isn't entirely there yet but I want to get your eyes on it. I have a few questions I'll call out in PR comments. One additional thing I'm not sure about - it seems like all of the tests are being recognized but skipped, but I don't see anything in the code dictating that anything should be skipped. This is happening both in Github Actions and when I run tests locally.

File: README.rst [hunk 1]
```diff
@@ -17,7 +19,110 @@ for compatibility purposes.
...
+-----------------------
+
+First, clone the repository and prep your Python environment:
+
+.. code:: bash
+
+    git clone https://github.com/theatlantic/django-select2-forms.git
+    pyenv install 3.6.8
+    pyenv install 3.7.2
+    pyenv install 3.8.0
```
Comment (lolasov) [inline README.rst:65]: Not sure how to know which exactly python versions to install for `3.8` and `3.9`

File: tox.ini [hunk 2]
```diff
@@ -1,22 +1,64 @@
...
+commands =
+    find {toxinidir} -type f -name "*.pyc" -delete
+    find {toxinidir} -type d -name "__pycache__" -delete
+    rm -f {toxinidir}/tests/db.sqlite {toxworkdir} {toxinidir}/.pytest_cache {toxinidir}/build
+
+[testenv:pep8]
+description = Run PEP8 flake8 against the select2/ package directory
+skipsdist = true
+skip_install = true
+basepython = python3.7
```
Comment (lolasov) [inline tox.ini:43]: Should this be 3.6?

File: tox.ini [hunk 3]
```diff
@@ -1,22 +1,64 @@
...
     django-selenosis
-    dj-database-url
-    dj111: Django~=1.11.0
-    dj20: Django~=2.0.0
-    dj111-grp: django-grappelli==2.10.2
-    dj20-grp: django-grappelli==2.10.2
+    django22: Django>=2.2,<3.0
+    django31: Django>=3.1,<3.2
+    django32: Django>=3.2b1,<4.0
+    django22-grp: django-grappelli==2.13.4
```
Comment (lolasov) [inline tox.ini:22]: I am not sure if these `django-grappelli` versions are correct - I just went off the releases listed [here](https://pypi.org/project/django-grappelli/)

File: tox.ini (hunk 2 above)
Comment (dakrauth) [reply tox.ini:43]: Let's default to 3.7. Elsewhere we have removed 3.6, so we can do so here as well.

File: tox.ini (hunk 3 above)
Comment (dakrauth) [reply tox.ini:22]: Should be fine, I believe we did so similarly in another package

File: README.rst [hunk 4]
```diff
@@ -17,7 +19,110 @@ for compatibility purposes.
 
 * ~=v2.0.2: Python ~=2.7,~=3.6 | Django >=1.8,<2.1
 * ~=v2.1: Python ~=2.7,>=3.6,<3.8 | Django >=1.11,<2.1
-* ~=v3.0: __Python >=3.6,<3.9 | Django >=2.0,<2.1 (future release)__
+* ~=v3.0: Python >=3.6,<3.9 | Django 2.2,3.1,3.2 (current release)
```
Comment (dakrauth) [inline README.rst:22]: You can remove the 1st 3 bullet points and leave just this line

**Review by dakrauth — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (dakrauth) [reply README.rst:62]: You can check with `pyenv install -l | grep " 3\.[89]\.\d+$"`

**Review by dakrauth — APPROVED** [review]
Looks good!

**Pull request merged**